### PR TITLE
Issue #2746 DC Certificate

### PR DESCRIPTION
**Pull request by j0rt3g4**
Issue #2746 DC Certificate template clarifications

### CODE REVIEW

**Comment by mypil** [comment]
@TokyoScarab - kindly do the copy editor review for this PR. Thanks.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md [hunk 1]
```diff
@@ -68,13 +68,19 @@ Certificate authorities write CRL distribution points in certificates as they ar
...
 
 - The domain controller has the private key for the certificate provided.
 - The root CA that issued the domain controller's certificate is in the device's **Trusted Root Certificate Authorities**. 
+- Use the **Kerberos Authentication certificate template** instead of any other older template.
 - The domain controller's certificate has the **KDC Authentication** enhanced key usage.
 - The domain controller's certificate's subject alternate name has a DNS Name that matches the name of the domain.
 
+
+> [!Tip]
+> If you are using windows server 2008, **Kerberos Authentication** is not the default template, so make sure to use the correct template when issuing/re-issuing the certificate.
```
Comment (TokyoScarab) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md]:
```suggestion
> If you are using Windows Server 2008, **Kerberos Authentication** is not the default template, so make sure to use the correct template when issuing or re-issuing the certificate.
```

**Comment by j0rt3g4** [comment]
@TokyoScarab  Thank you! it's done.

File: windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md [hunk 2]
```diff
@@ -68,13 +68,19 @@ Certificate authorities write CRL distribution points in certificates as they ar
 
 #### Why does Windows need to validate the domain controller certifcate?
```
Comment (mypil) [inline windows/security/identity-protection/hello-for-business/hello-hybrid-aadj-sso-base.md]:
```suggestion
#### Why does Windows need to validate the domain controller certificate?
```

**Comment by mypil** [comment]
@j0rt3g4 - can you also correct the typo error on line 167/173 specific to( ### Configure the new CRL distribution point and Publishing location in the issuing _certifcate_ authority)? This should be "certificate". Thanks.

**Comment by j0rt3g4** [comment]
@mypil  done

**Comment by mypil** [comment]
@mikestephens-MS - can you please assist to check, approve and merge the content for this PR based on issue #2746?

Thank you.

**Comment by DaniHalfin** [comment]
Hi @mypil, Mike switched teams so please address anything he authored to me (for now).
Checking for this one.

**Comment by DaniHalfin** [comment]
@mapalko is looking into this one.

**Comment by mapalko** [comment]
The change looks good.

**Comment by DaniHalfin** [comment]
@mypil I'm merging.

**Comment by mypil** [comment]
@DaniHalfin - thank you

**Pull request merged**